libraries/AP_Baro/AP_Baro_MS5611.h

```diff
@@ -28,6 +28,10 @@
 #define HAL_BARO_MS5637_I2C_ADDR 0x76
 #endif
 
+#if AP_BARO_MS5837_ENABLED
+#define MS5837_30BA_02BA_SELECTION_THRESHOLD 37000
```

Comment:
```suggestion
// Determined in https://github.com/ArduPilot/ardupilot/pull/29122#issuecomment-2877269114
#define MS5837_30BA_02BA_SELECTION_THRESHOLD 37000
```
Might be worth including some context for the value 🤷‍♂️